_locales/en/messages.json

```diff
@@ -119,9 +119,19 @@
 		"message": "No tabs available to save",
 		"description": "Alert dialog message when there's no tabs to save"
 	},
+	"errorSavingTabs":
```

Comment:
I'm not sure how to handle localization, I don't know any of the other languages available in the extension, so I didn't add the new strings in the other locales files. Should we ping the contributors who made the other localizations to ask for their help ?